src/AI/AIUtils.java

```diff
@@ -167,4 +168,24 @@ public static GameAction moveTowardLocation(Unit unit, XYCoord destination, Game
     }
     return stations;
   }
+  
+  /**
+   * Find a usable ability that has all specified flags, and add it to the provided queue.
+   */
+  public static CommanderAbility queueCromulentAbility(Queue<GameAction> q, Commander co, int flags)
```

Comment:
ugh... I'm not sure I can countenance this.

Reply:
Cromulent is a totally cromulent word.